server/api/src/rsDataObjOpen.cpp

```diff
@@ -220,6 +223,10 @@ namespace
                 return 0;
             }
 
+            irods::log(LOG_DEBUG, fmt::format(
+                "[{}:{}] - creating a new replica [path=[{}], hierarchy=[{}]",
```

Comment:
add missing second closing bracket at end of hierarchy

Reply:
Done